### ISSUE TITLE
Rom list cache doesn't work on Windows version.

### ISSUE THREAD

**Issue by McCloud1710** [Rosalie241/RMG#46]
Every time I open RMG the rom list reload from scratch, turning the start slower. The linux version works fine and the list shows up instantaneously.

**Comment by Xii-Nyth** [comment]
hmm its fine for me, is the folder its in set to read only?

**Comment by Xii-Nyth** [comment]
huh weird. where is you roms folder located?

**Comment by McCloud1710** [comment]
I keep all my roms on a second HDD(not a external one), so I can use my emulators in both systems(I have W10 and Opensuse installed). The path is something like _G:\Games\Emul\Roms\Nintendo\N64_ . 

Fun thing is that the emulator remember the rom path and load the roms without issues. It just don't keep the roms cache.

**Comment by Xii-Nyth** [comment]
Oh ok then, was just making sure it wasnt on the first directory of the drive like C:\the roms or it sometimes has issues with permissions. That being said sometimes permissions get corrupted and will ahve to delete them and re add them.

**Comment by Tasosgemah** [comment]
I have the same issue. I have the roms in G:\Emulators\N64\Roms and it's not a read only folder.

**Comment by Rosalie241** [comment]
Can you try again with v0.1.9?

**Comment by Tasosgemah** [comment]
> Can you try again with v0.1.9?

Works now for me. Though, at the end of some "internal name" entries i'm seeing a question mark inside a square shape . Not a big deal though.

**Comment by Rosalie241** [comment]
Can you show me a screenshot?

**Comment by Tasosgemah** [comment]
> Can you show me a screenshot?

![Untitled](https://user-images.githubusercontent.com/12124465/202875776-503caa1b-d506-4547-8464-835212973f14.png)

It's random, sometimes when i open the emulator the names are shown correctly, other times they show this. It's about 50/50 chance.

**Comment by Rosalie241** [comment]
Does https://github.com/Rosalie241/RMG/actions/runs/3508221216 fix it? (it'll take 20 mins to build from now so no rush in testing)

**Comment by Tasosgemah** [comment]
Yeah, looks like it's fixed, couldn't reproduce those artifacts so far.

**Comment by Rosalie241** [comment]
it's been pushed as https://github.com/Rosalie241/RMG/commit/878f79e0477aab29d0a1a1d086a04ea6b8c57584, will be included in the next release :heart:, thank you for the report & help!

**Comment by McCloud1710** [comment]
Greetings. And the issue is gone. 

I have installed the new 0,1,9 and now the cache works fine on Windows too.

Thanks for fixing it :) .
